### PR TITLE
fix: hide MT merge option if without URL

### PR DESCRIPTION
**Pull request by aeng**
https://zanata.atlassian.net/browse/ZNTA-2743

JIRA issue URL: *paste URL here*

*[description]*

## Checklist

- JIRA link
- Check target branch
- Make sure all commit statuses are green or otherwise documented reasons to ignore
- QA needs to evaluate against the JIRA ticket
- Changed files and commits make sense for this PR

See [Zanata Development Guidelines](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines) more for information.

----
*This template can be updated in .github/PULL_REQUEST_TEMPLATE.md*

### CODE REVIEW

File: server/services/src/main/java/org/zanata/action/VersionHomeAction.java [hunk 1]
```diff
@@ -1402,4 +1416,8 @@ public CopyVersionHandler getCopyVersionHandler() {
     public AbstractListFilter<HLocale> getDocumentsTabLanguageFilter() {
         return this.documentsTabLanguageFilter;
     }
+
+    public boolean isMTEnabled() {
+        return identity.hasRole("mt-bulk") && mtServiceURL != null;
```
Comment (djansen-redhat) [inline server/services/src/main/java/org/zanata/action/VersionHomeAction.java]: Should the order be reversed? (Not blocking, just a thought)

Comment (aeng) [reply server/services/src/main/java/org/zanata/action/VersionHomeAction.java]: Yup. Done

File: server/services/src/main/java/org/zanata/action/VersionHomeAction.java [hunk 2]
```diff
@@ -163,6 +167,16 @@
     private WebhookServiceImpl webhookService;
     @Inject
     private WebHookDAO webHookDAO;
+    @Inject
+    @MTServiceURL
+    private URI mtServiceURL;
+    @Inject
+    @MTServiceUser
```
Comment (djansen-redhat) [inline server/services/src/main/java/org/zanata/action/VersionHomeAction.java]: Are these used?

Comment (aeng) [reply server/services/src/main/java/org/zanata/action/VersionHomeAction.java]: Cleaned up.

**Comment by djansen-redhat** [comment]
___
*[server/zanata-war/src/main/webapp/iteration/view.xhtml, line 428 at r3](https://reviewable.io/reviews/zanata/zanata-platform/1018#-LJMGRfV4w66ZtYn1Z0V:-LJMGRfWBwaxrL3PwfOL:b-i7fsjh) ([raw file](https://github.com/zanata/zanata-platform/blob/2971b192dc3b8d8017609d6e15d2e042e662db58/server/zanata-war/src/main/webapp/iteration/view.xhtml#L428)):*
> ```HTML
>                       <ui:fragment rendered="#{versionHomeAction.isMTEnabled()}">
>                         <li>
>                           #{systemPropertyConfigStore.isOAuthEnabled()}
> ```

What's this one do?

**Comment by djansen-redhat** [comment]
Verified

**Pull request merged**